src/index.js

```diff
@@ -1,5 +1,19 @@
 import './style.css';
+import addNewScore from './modules/score.js';
 
-import displayContent from './modules/score.js';
+import {
+  refreshButton, nameInputText, scoreInputText, submitScore,
+} from './modules/domVariable.js';
+import createGame from './modules/createGame.js';
+import submitData from './modules/sendData.js';
 
-displayContent();
\ No newline at end of file
+submitScore.addEventListener('click', () => {
+  const Name = nameInputText.value;
+  const Score = scoreInputText.value;
+  submitData(Name, Score);
```

Comment:
- [ Optional ] I think it would be better if you `refresh` the UI immediately after the `submit` button has been clicked instead of the user having to manually refresh the scoreboard by clicking the `refresh` button. Auto-refreshing the UI will give a good User Experience to users, kindly consider doing this.